pages/start.tsx

```diff
@@ -153,16 +160,36 @@ const LoginStep2: React.FC<LoginStep2Props> = ({ handleLoginStep, setRepoList })
         });
 
         setRepoList(repoList);
+        setAddOrgError(OrgLookupError.Initial);
       }
 
       // If valid, go to next step
       handleLoginStep();
-    } catch (e) {
+    } catch (e: any) {
```

Comment:
please remove the `any` and console logging here, provide a user alert if the message is important to the user

Reply:
Console logging was there, I was confused about whether it is intended or not.

If I remove 'any' it does not allow me to access the "e.code" since it was required to get the error code. The error message appears to the user based on the error code.

Reply:
catching infers Error; when you are throwing objects that are more than Error, you should type it more strictly, not less IE any.

please type e to an actua error and ensure only that can be thrown 🍕